src/control/server/storage/faults.go

```diff
@@ -60,15 +60,23 @@ func FaultScmInvalidPMem(msg string) *fault.Fault {
 }
 
 // FaultScmNoModules represents an error where no PMem modules exist.
-var FaultScmNoModules = storageFault(code.ScmNoModules,
-	"No PMem modules exist on storage server",
-	"Install PMem modules and retry command")
-
-// FaultBdevConfigTypeMismatch represents an error where an incompatible mix of emulated and
+var FaultScmNoModules = storageFault(
+	code.ScmNoModules,
+	"no PMem modules exist on storage server",
+	"install PMem modules and retry command")
+
+// FaultScmConfigTierMissing creates a Fault when no scm tier is present in engine storage config.
+var FaultScmConfigTierMissing = storageFault(
+	code.ScmConfigTierMissing,
+	"missing scm storage tier in engine storage config",
+	"add a scm tier at index-0 in engine storage tiers list in server config file and restart daos_server")
```

Comment:
Just a thought for clarity, instead of saying "at index 0," how about "in the first position"? The indices may not be obvious to a user writing a config file.

Reply:
done